algorithms/backtrack/maxsum_subarray.py

```diff
@@ -0,0 +1,48 @@
+#time complexity : O(nlogn)
+#space complexity : O(n)
+
+# Returns sum of maximum sum subarray in arr[low.....high]
+def maxSumSubArray(arr, low, high) :
+    # Base Case
+    if (low == high) :
+        return arr[low]
+    # Find middle element
+    mid = low + ((high - low) // 2)
+    # Return maximum of left ,right and mid crossing
+    return max(maxSumSubArray(arr, low, mid),
+               maxSumSubArray(arr, mid+1, high),
+               maxSumCrossing(arr, low, mid, high))
+
+# Find the maximum possible sum in arr[] such that arr[m] is part of it
+def maxSumCrossing(arr, low, mid, high) :
+    # Include elements on left of mid.
+    summ = 0; left_sum = -10000
```

Comment:
is `-10000` representing a very small number? If so, could you change it to `float("-inf")`.

Reply:
I have added the proposed updations.